pkg/tarsum/tarsum_test.go

```diff
@@ -138,11 +138,12 @@ func sizedTar(opts sizedOptions) io.Reader {
 	defer tarW.Close()
 	for i := int64(0); i < opts.num; i++ {
 		err := tarW.WriteHeader(&tar.Header{
-			Name: fmt.Sprintf("/testdata%d", i),
-			Mode: 0755,
-			Uid:  0,
-			Gid:  0,
-			Size: opts.size,
+			Name:     fmt.Sprintf("/testdata%d", i),
+			Mode:     0755,
+			Uid:      0,
+			Gid:      0,
+			Size:     opts.size,
+			Typeflag: tar.TypeReg,
```

Comment:
This is the change that makes Go 1.10 switch to the same behavior as Go 1.11? With this change, do we still need the vendored `archive` package (on Go 1.10)?

Reply:
> This is the change that makes Go 1.10 switch to the same behavior as Go 1.11?

Yes. It just fixes the test case by explicitly specifying the file type (rather than using an obsolete default value).

> With this change, do we still need the vendored archive package (on Go 1.10)?

Yes we do (it is there to fix a different problem about static build).